server/db.ts

```diff
@@ -0,0 +1,90 @@
+import dotenv from 'dotenv';
+import { createConnection } from 'typeorm';
+import { MinLength, Matches, IsBoolean, IsNotEmpty, Min, validate } from 'class-validator';
+import { Expose, plainToClass, Type, Transform } from 'class-transformer';
+import { MysqlConnectionOptions } from 'typeorm/driver/mysql/MysqlConnectionOptions';
+
+export class Environment {
```

Comment:
All environment variable relate code should be in own file 👍 This file should only include stuff for database connection